### ISSUE TITLE
Automatic Message Encryption with DLP Policies

### ISSUE THREAD

**Issue by ASM-bler** [MicrosoftDocs/OfficeDocs-o365seccomp#62]
"Can I automatically encrypt messages by setting up policies in Data Loss Prevention (DLP) through the Security & Compliance Center?
Currently you can only set up mail flow rules in Exchange Online. Encryption is currently not supported in DLP through the Security & Compliance Center."

I don't think this is still the case. I was able to automatically encrypt messages by setting up a DLP Policy in the Security & Compliance Center.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 57ad427d-d869-2ab6-3d2c-f846efc99256
* Version Independent ID: 606eaa37-2713-8087-015f-012ae9c90090
* Content: [Office 365 Message Encryption FAQ](https://docs.microsoft.com/en-us/office365/securitycompliance/ome-faq)
* Content Source: [SecurityCompliance/ome-faq.md](https://github.com/microsoftdocs/officedocs-o365seccomp/blob/public/SecurityCompliance/ome-faq.md)
* Service: **o365-administration**
* GitHub Login: @KCCross
* Microsoft Alias: **krowley**

**Comment by denisebmsft** [comment]
Thank you for the feedback, @ASM-bler. We will look into this.

**Comment by AndreaBarr** [comment]
@ASM-bler Thank you for this feedback

I will get this to the o365seccomp team for investigation and updating. Thank you for taking the time to make the docs better. 

I was able to get this into the backlog for the O365seccomp team. The team will get this prioritized into a sprint plan. Thank you for reporting and making the docs better. Much appreciated. 
 
I made a note to request the team to update this when the work is complete.
